exercises/list-ops/src/test/java/ListOpsTest.java

```diff
@@ -0,0 +1,473 @@
+                );
+        final List<Integer> actual = ListOps.concat(list);
+
+        assertNotNull(actual);
+        assertFalse(actual.isEmpty());
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @Ignore
+    public void shouldConcatenateOneEmptyList() {
+        final List<Integer> actual = ListOps.concat(EMPTY_LIST);
+
+        assertNotNull(actual);
+        assertTrue(actual.isEmpty());
+    }
+
+    @Test
+    @Ignore
+    public void shouldConcatenateTwoEmptyLists() {
+        final List<Integer> actual = ListOps.concat(EMPTY_LIST, EMPTY_LIST);
+
+        assertNotNull(actual);
+        assertTrue(actual.isEmpty());
+    }
+
+    @Test
+    @Ignore
+    public void shouldConcatenateOneEmptyAndOneNonEmptyLists() {
+        final List<Integer> expected
+                = Collections.unmodifiableList(
+                        IntStream.range(0, 4)
+                                .boxed()
+                                .collect(Collectors.toList())
+                );
+        final List<Integer> list
+                = Collections.unmodifiableList(
+                        IntStream.range(0, 4)
+                                .boxed()
+                                .collect(Collectors.toList())
+                );
+        final List<Integer> actual = ListOps.concat(list, EMPTY_LIST);
+
+        assertNotNull(actual);
+        assertFalse(actual.isEmpty());
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @Ignore
+    public void shouldConcatenateOneNonEmptyAndOneEmptyLists() {
+        final List<Integer> expected
+                = Collections.unmodifiableList(
+                        IntStream.range(0, 4)
+                                .boxed()
+                                .collect(Collectors.toList())
+                );
+        final List<Integer> list
+                = Collections.unmodifiableList(
+                        IntStream.range(0, 4)
+                                .boxed()
+                                .collect(Collectors.toList())
+                );
+        final List<Integer> actual = ListOps.concat(EMPTY_LIST, list);
+
+        assertNotNull(actual);
+        assertFalse(actual.isEmpty());
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @Ignore
+    public void shouldConcatenateTwoListsWithSameElements() {
+        final List<Integer> list1 = Collections.unmodifiableList(
+                IntStream.range(0, 4).boxed().collect(Collectors.toList())
+        );
+        final List<Integer> list2 = Collections.unmodifiableList(
+                IntStream.range(1, 6).boxed().collect(Collectors.toList())
+        );
+        final List<Integer> sortedCancatenatedList
+                = Stream.concat(list1.stream(), list2.stream())
+                        .collect(Collectors.toList());
+        Collections.sort(sortedCancatenatedList);
+        final List<Integer> expected = Collections.unmodifiableList(
```

Comment:
Otherwise, didn't touch the code. Lists to concatenate are used both to create the result and to test the _concat()_ method. Could you take a look again here and elaborate more, please.